recipes_source/recipes/what_is_state_dict.py

```diff
@@ -1,30 +1,31 @@
 """
-What is a state_dict in PyTorch
-===============================
-In PyTorch, the learnable parameters (i.e. weights and biases) of a
-``torch.nn.Module`` model are contained in the model’s parameters
-(accessed with ``model.parameters()``). A ``state_dict`` is simply a
-Python dictionary object that maps each layer to its parameter tensor.
-
-Introduction
+PyTorch에서 state_dict란 무엇인가요?
+======================================
+PyTorch에서 ``torch.nn.Module`` 모델의 학습 가능한
+매개변수(예. 가중치와 편향)들은 모델의 매개변수에 포함되어 있습니다.
+(model.parameters()로 접근합니다)
+``state_dict`` 는 간단히 말해 각 계층을 매개변수 텐서로 매핑되는
+Python 사전(dict) 객체입니다.
+
+개요
 ------------
-A ``state_dict`` is an integral entity if you are interested in saving
-or loading models from PyTorch.
-Because ``state_dict`` objects are Python dictionaries, they can be
-easily saved, updated, altered, and restored, adding a great deal of
-modularity to PyTorch models and optimizers.
-Note that only layers with learnable parameters (convolutional layers,
-linear layers, etc.) and registered buffers (batchnorm’s running_mean)
-have entries in the model’s ``state_dict``. Optimizer objects
-(``torch.optim``) also have a ``state_dict``, which contains information
-about the optimizer’s state, as well as the hyperparameters used.
-In this recipe, we will see how ``state_dict`` is used with a simple
-model.
-
-Setup
------
-Before we begin, we need to install ``torch`` if it isn’t already
-available.
+``state_dict`` 는 PyTorch에서 모델을 저장하거나 불러오는 데 관심이
+있다면 필수적인 항목입니다.
+``state_dict`` 객체는 Python 사전이기 때문에 쉽게 저장, 업데이트,
+변경 및 복원할 수 있으며, PyTorch 모델과 옵티마이저에 엄청난
+모듈성(modularity)을 제공합니다.
+이 때, 학습 가능한 매개변수를 갖는 계층(합성곱 계층, 선형 계층 등)
+및 등록된 버퍼들(batchnorm의 running_mean)이 모델의 ``state_dict``
+에 항목을 가진다는 점에 유의하시기 바랍니다. 옵티마이저 객체
```

Comment:
```suggestion
및 등록된 버퍼들(batchnorm의 running_mean)만 모델의 ``state_dict``
 항목을 가진다는 점에 유의하시기 바랍니다. 옵티마이저 객체
```
원문에 Only라는 글자가 있으므로, 학습 가능한 매개변수와 등록된 버퍼들만 항목을 가진다는 의미로 해석됩니다.